weasyl/test/test_login.py

```diff
@@ -0,0 +1,598 @@
+# This Python file uses the following encoding: utf-8
+import unittest
+import pytest
+import py
+import bcrypt
+import arrow
+import web
+import os
+import json
+
+import db_utils
+from weasyl import login, macro
+from weasyl import define as d
+from weasyl.error import WeasylError
+from _pytest import tmpdir
+
+# Main test account
+raw_password = "0123456789"
+old_2a_bcrypt_hash = "$2a$12$qReI924/8pAsoHu6aRTX2ejyujAZ/9FiOOtrjczBIwf8wqXAJ22N."
+
+
+"""
+Section containing functions used within this suite of tests.
+"""
+TestAccountName_nameprefix = "testlogin"
+TestAccountName_counter = 0
+class TestFunctions():
```

Comment:
Deterministic tests are nice; could you use the same account name and password for every test rather than generating them randomly, please?